drivers/gles3/rasterizer_gles3.cpp

```diff
@@ -29,6 +29,7 @@
 /**************************************************************************/
 
 #include "rasterizer_gles3.h"
+#include "servers/rendering/rendering_device.h"
```

Comment:
```suggestion
```
Already included

Reply:
Zed's autocompletion (likely coming from clangd) that is applied when using format-on-save gets too eager to include those :slightly_frowning_face: